src/components/Text/text.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../colors';
+
+.heading-1,
+.heading-2,
+.heading-3,
+.heading-4,
+.heading-5,
+.heading-6,
+.paragraph-1,
+.paragraph-2,
+.label {
+  font-family: 'IBM Plex Sans', sans-serif;
+  font-style: normal;
+  margin: 0;
+
+  &--primary {
+    color: $primary;
+  }
+
+  &--secondary {
+    color: $secondary;
+  }
+
+  &--muted {
+    color: $muted;
+  }
+
+  &--accent {
+    color: $accent;
+  }
+
+  &--warning {
+    color: $warning;
+  }
+
+  &--success {
+    color: $success;
+  }
+
+  &--error {
+    color: $error;
+  }
+
+  &--bright {
+    color: $bright;
+  }
+}
+
+.heading-1,
+.heading-2,
+.heading-3,
+.heading-4,
+.heading-5,
+.heading-6 {
+  font-weight: 600;
+}
+
+.paragraph-1,
+.paragraph-2,
+.label {
+  font-weight: 400;
+}
+
+.heading-1 {
+  font-size: 72px;
+}
```

Comment:
Those size's can be `subclass` of `heading` main class as we discussed :) It will make code shorter and more readable.